crates/ruff_server/src/edit.rs

```diff
@@ -139,6 +139,7 @@ impl WorkspaceEditTracker {
                 document_edits.push(lsp_types::TextDocumentEdit {
                     text_document: lsp_types::OptionalVersionedTextDocumentIdentifier {
                         uri,
+                        // TODO(jane): Re-enable versioned edits after investigating whether it could work with notebook cells
                         version: None,
```

Comment:
I think we do have it enabled in `ruff-lsp`. What kind of problem are we facing here? If it's not critical, it might be useful to open an issue as well.

Reply:
The issue here is that we pass in the notebook document version and use it as the cell version. The notebook API isn't set up yet to track the versions of individual cell documents and I'd rather handle this in a follow-up.